source/pmc/responsibilities.md

```diff
@@ -0,0 +1,88 @@
+---
+title: PMC Responsibilities
+---
+
+The Project Management Committee (PMC) is responsible for the oversight
+of the project, including technical decisions, ensuring that the project
+is operating in accordance with ASF norms, adding new members to the
+project, and voting on releases.
+
+* [Conducting business](#conducting-business)
+* [Ensuring project health](#ensuring-project-health)
+* [Adding community members](#adding-community-members)
+* [Reporting](/pmc/reporting)
+
+## Conducting Business
+
+The PMC is expected to conduct all of its business on the public
+developers mailing list, in the full view of the community.
+
+The private@ list is for communication about private matters, such as
+discussion of proposed committers, proposed new PMC members, and
+communication from the Board of Directors. Do not use it for discussion
+of features, roadmap, or other technical decisions.
+
+The Board of Directors will occasionally send you comments or queries,
+often related to your quarterly board report. Any PMC member can, and
+should, respond to these. You should not wait for the Chair to respond
+to these queries.
+
+The PMC is tasked with making decisions on the direction of the project,
+They can, and should, also incorporate the voice of the larger community
+in these decisions. However, if a vote is taken, only the votes of the
+PMC are binding on the outcome. Other community members may vote as
+well, but those votes are advisory only, and not binding.
+
+A PMC member may veto a technical decision, such as a proposed patch.
```

Comment:
Comment here. I am not sure how it is in other projects, but we never thought PMC members have more "voting"  or "veto" rights in case  of technical decisions than committers have. At least in our cases PMC members have no more power to decide if a certain - even big - change in the direction of the project should happen. The only "binding" decisions that only PMC members have are release votes, all other votes in our case - including voting on big "Airflow Improvement Proposals", commiters have binding votes, and also committers have veto powers for those. 

I wonder if this is normal/expected - in the light of this comment, I believe it  could be up to the project to decide who has the voting powers and it is also - I think - reflected here https://www.apache.org/foundation/voting.html 

So maybe worth to mention that thos voting powers can be also - if the PMC members decide - passed to regular committers?

Or maybe we are doing it wrongly ?